bip-taro-addr.mediawiki

```diff
@@ -54,7 +54,38 @@ We refer to this value as the <code>taro_hrp</code>
 Given the 32-byte <code>asset_id</code>, 32-byte
```

Comment:
Could reword to clarify that all keys need to be 32-byte x-only BIP 340/341 pubkeys, not just the internal key.